explorer/explorer.go

```diff
@@ -295,15 +296,21 @@ func New(dataSource explorerDataSource, userRealIP bool) *explorerUI {
 			t, _ := time.Now().Zone()
 			return t
 		},
-		"float64SplitZeroSuffix": func(v float64) []string {
+		"float64AsDecimalParts": func(v float64, useCommas bool) []string {
 			clipped := fmt.Sprintf("%.8f", v)
 			oldLength := len(clipped)
 			clipped = strings.TrimRight(clipped, "0")
 			trailingZeros := strings.Repeat("0", oldLength-len(clipped))
-			result := append(make([]string, 0, 2), clipped, trailingZeros)
-			return result
+			valueChunks := strings.Split(clipped, ".")
+			integer := valueChunks[0]
+			if useCommas {
+				integerAsInt64, _ := strconv.ParseInt(integer, 10, 64)
+				integer = humanize.Comma(integerAsInt64)
+			}
+			dec := valueChunks[1]
```

Comment:
this could panic if valueChunks length <2.  I don't see how that could happen easily, but it would be best to guard for this.